pypeit/reduce.py

```diff
@@ -1109,10 +1113,15 @@ def find_objects_pypeline(self, image, std_trace=None,
             msgs.info(qa_title)
             thismask = self.slitmask == slit_spat
             inmask = self.sciImg.select_flag(invert=True) & thismask
-            # Find objects
             specobj_dict = {'SLITID': slit_spat,
-                            'DET': self.det, 'OBJTYPE': self.objtype,
+                            'OBJTYPE': self.objtype,
                             'PYPELINE': self.pypeline}
+            if isinstance(self.det, tuple):
+                # Image is a detector mosaic
+                specobj_dict['DET'] = self.spectrograph.allowed_mosaics.index(self.det)+1
+                specobj_dict['from_mosaic'] = True
```

Comment:
Does from_mosaic get used? Is the goal here to propagate something into specobjs?